examples/svelte/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "svelte-example",
+	"version": "0.0.1",
```

Comment:
Note that this will be published to `latest` at version `0.0.1` when we do our next release. Is this desirable? We can consider publishing to `@preview` until this is production ready like we did for `@ui-react`.

cc @Milan-Shah @ErikCH

Reply:
Oops, meant to post this comment to `packages/svelte/package.json`

Reply:
OK, after our conversation yesterday, I'll resolve this. New packages will be pushed to a new branch `ui-svelte/main` and that branch will push to  a `next` release tag.